cmd/kubeadm/app/phases/kubelet/kubelet.go

```diff
@@ -39,6 +39,9 @@ import (
 
 // CreateBaseKubeletConfiguration creates base kubelet configuration for dynamic kubelet configuration feature.
 func CreateBaseKubeletConfiguration(cfg *kubeadmapi.MasterConfiguration, client clientset.Interface) error {
+	fmt.Println(fmt.Sprintf("[kubelet] Uploading a ConfigMap %q in namespace %s with base configuration for the kubelets in the cluster",
```

Comment:
you can write this as `fmt.Printf("[kubelet] Uploading a ConfigMap %q in namespace %s with base configuration for the kubelets in the cluster", kubeadmconstants.KubeletBaseConfigurationConfigMap, metav1.NamespaceSystem)` instead

Reply:
Ah! I don't know what I was thinking when I wrote this... :sob: